quarkus-cli/src/test/java/io/quarkus/ts/quarkus/cli/QuarkusCliCreateJvmApplicationIT.java

```diff
@@ -263,6 +263,7 @@ public void shouldAddAndRemoveExtensions() {
 
         // The health endpoint should be now gone
         app.restart();
+        app.restartAndWaitUntilServiceIsStarted();
```

Comment:
Shouldn't this replace `app.restart();` on the line above?

Reply:
done